Doc/library/ctypes.rst

```diff
@@ -2142,11 +2142,10 @@ Utility functions
 
 .. function:: WinError(code=None, descr=None)
 
-   This function is probably the worst-named thing in ctypes. It
-   creates an instance of :exc:`OSError`.  If *code* is not specified,
-   ``GetLastError`` is called to determine the error code. If *descr* is not
-   specified, :func:`FormatError` is called to get a textual description of the
-   error.
+   This function is probably the worst-named thing in ``ctypes``. It creates an
+   instance of :exc:`OSError`.  If *code* is not specified, :func:`GetLastError`
+   is called to determine the error code. If *descr* is not specified,
+   :func:`FormatError` is called to get a textual description of the error.
```

Comment:
Since Python 3.3 merged `WindowsError`   into `OSError`, this is no longer the worst-named thing in `ctypes` (if it ever was) :)
And the negativity isn't helping anyone.
```suggestion
   Creates an
   instance of :exc:`OSError`.  If *code* is not specified, :func:`GetLastError`
   is called to determine the error code. If *descr* is not specified,
   :func:`FormatError` is called to get a textual description of the error.
```